tests/test_metadata.py

```diff
@@ -143,3 +149,20 @@ def test_submit_metadata_manifest(
                     hide_blanks=hide_blanks,
                 )
                 assert mock_manifest_id == "mock manifest id"
+
+    def test_submit_filebased_manifest(self, helpers):
+        meta_data_model = metadata_model(helpers, "class_label")
+
+        manifest_path = helpers.get_data_path(
+            "mock_manifests/filepath_submission_test_manifest.csv"
+        )
+
+        with does_not_raise():
+            manifest_id = meta_data_model.submit_metadata_manifest(
+                manifest_path=manifest_path,
+                dataset_id="syn62276880",
+                manifest_record_type="file_only",
+                restrict_rules=False,
+                file_annotations_upload=True,
+            )
+            assert manifest_id == "syn62280543"
```

Comment:
Nit (nitpicky): Just so I'm understanding, is this an integration test?  If so, since we know we want to make those "quality of life" changes, I'd go ahead and start adding tests in this format

```
.
└── tests
    ├── data
    │   ├── mock_manifests
    │   └── test_configs
    ├── integration
			data (Moved from above)
			test_configs (Moved from above)
    └── unit
			data (Moved from above)
			test_configs (Moved from above)
```

Without worrying about the older tests.  Just a thought.